third_party/bigframes_vendored/sklearn/preprocessing/_imputation.py

```diff
@@ -0,0 +1,44 @@
+# Authors: Nicolas Tresegnie <nicolas.tresegnie@gmail.com>
+# License: BSD 3 clause
+
+from bigframes_vendored.sklearn.base import BaseEstimator, TransformerMixin
+
+from bigframes import constants
+
+
+class Imputer(BaseEstimator, TransformerMixin):
```

Comment:
Where does this come from?

The counter part should be. https://scikit-learn.org/stable/modules/generated/sklearn.impute.SimpleImputer.html#sklearn.impute.SimpleImputer

Reply:
Should I be `sklearn.preprocessing.Imputer`: https://scikit-learn.org/0.18/modules/generated/sklearn.preprocessing.Imputer.html?

Reply:
"`SimpleImputer` replaces the previous `sklearn.preprocessing.Imputer`
       estimator which is now removed." Saw it. Addressed.